src/server/.tests/routes/controllers/Middlewares/logInUser.spec.js

```diff
@@ -14,7 +14,7 @@ describe('logInUser', () => {
 		suite.authenticatio
 
 	})
-	it('should call next() without calling authenticationMiddleware, when req.isAuthenticated is true', () => {
+	it('should terminate this login middleware by calling next() when user is already logged in ', () => {
```

Comment:
`should skip to next middleware` terminate sounds like it exits thread from OS POV.